src/runner/Resources.resx

```diff
@@ -89,7 +148,7 @@
     <value>This setting has been disabled by your administrator.</value>
   </data>
   <data name="STARTUP_DISABLED_BY_USER" xml:space="preserve">
-    <value>This setting has been disabled manually via &lt;a href=&quot;https://ms_settings_startupapps&quot; target=&quot;_blank&quot;&gt;Startup Settings&lt;/a&gt;.</value>
+    <value>This setting has been disabled manually via &lt;a href="https://ms_settings_startupapps" target="_blank"&gt;Startup Settings&lt;/a&gt;.</value>
```

Comment:
Is this change working? Looks like unneeded change with the context of the PR.

Reply:
That's all done auto by Visual Studio.

Reply:
this needs to be validated it works if it was changed.

Reply:
@jaimecbernardo @crutkas If I interpret this result correctly, this string isn't even used anywhere. Does that answer your questions? 😆
![image](https://github.com/microsoft/PowerToys/assets/65828559/c0c7f0cc-cdab-4d19-91b1-f95f655e3b41)

Reply:
As it has a href, this is legacy then from the react style app that never got refactored out when that subsystem was removed

Reply:
Done